### PR TITLE
MTV-1279 - MTV 2.6.4 release notes

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTV-1279](https://issues.redhat.com/browse/MTV-1279)

### PREVIEW

* [ New features and enhancements 2.6.4](https://anarnold97.github.io/MTA-Preview/MTV/mtv-2-6-4-rn.html#new-features-and-enhancements-26-4_release-notes)
* [2.6.4 resolved issues](https://anarnold97.github.io/MTA-Preview/MTV/mtv-2-6-4-rn.html#resolved-issues-26-4_release-notes)
* [2.6 Known issues](https://anarnold97.github.io/MTA-Preview/MTV/mtv-2-6-4-rn.html#known-issues-26_release-notes)

### CONTENT

the release notes need to cover the following changes in 2.6.4:

* [MTV-1286](https://issues.redhat.com/browse/MTV-1286) - documented as a resolved issue

* [MTV-1299](https://issues.redhat.com/browse/MTV-1299) - documented as a resolved issue

* [MTV-1325](https://issues.redhat.com/browse/MTV-1325) - documented as a resolved issue

* [MTV-1344](https://issues.redhat.com/browse/MTV-1344) - documented as a known issue.

### CODE REVIEW

File: documentation/modules/rn-2.6.adoc [hunk 1]
```diff
@@ -123,72 +136,29 @@ With this update, you can edit plans that have failed to migrate any VMs. Some p
...
-.Migrating VMs with independent persistent disks from VMware to OCP-V fails
+[id="resolved-issues-26_{context}"]
+== Resolved issues
 
-Migrating VMs with independent persistent disks from VMware to OCP-V fails. link:https://issues.redhat.com/browse/MTV-993[(MTV-993)]
+This release has the following resolved issues:
 
-.Guest operating system from vSphere might be missing
+[id="resolved-issues-26-4_{context}"]
+=== Resolved issues 2.6.4
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:145]: We are also fixing

https://issues.redhat.com/browse/MTV-1299

Please add a section for this.

File: documentation/modules/rn-2.6.adoc [hunk 2]
```diff
@@ -317,3 +297,106 @@ In earlier releases of {project-short}, an earlier failed migration could have l
...
 
 For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12436210[Resolved Issues] in Jira.
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+
+.Disks and drives are offline after migrating Windows virtual machines from RHV or VMware to OCP
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:307]: This bug is actually fixed in 2.6.4!

But certain disks (windows "dynamic disks") are still impacted, see https://issues.redhat.com/browse/MTV-1344

**Review by fabiand — CHANGES_REQUESTED** [review]

File: documentation/modules/rn-2.6.adoc (hunk 1 above)
Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc:145]: MTV-1299 added to Resolved Issues 

| |
| --- |
| image |

Are we ok to keep the workaround?

Comment (fabiand) [reply documentation/modules/rn-2.6.adoc:145]: No, please remove it

Comment (fabiand) [reply documentation/modules/rn-2.6.adoc:145]: … because we did not focus on it in testing.

Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc:145]: Removed

File: documentation/modules/rn-2.6.adoc [hunk 3]
```diff
@@ -123,72 +136,73 @@ With this update, you can edit plans that have failed to migrate any VMs. Some p
...
+This release has the following resolved issues:
 
-.VMs from vSphere with RHEL 9 guest operating system can start with network interfaces that are down
+[id="resolved-issues-26-4_{context}"]
+=== Resolved issues 2.6.4
 
-When migrating VMs that are installed with RHEL 9 as a guest operating system from vSphere, the network interfaces of the VMs could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
+.Disks and drives are offline after migrating Windows virtual machines from RHV or VMware to OCP
 
+Windows (Windows 2022) VMs configured with multiple disks, which are *Online* before the migration, are *Offline* after a successful migration from {rhv-short} or VMware to {ocp}, using {project-short}. Only the `C:\` primary disk is *Online*. This issue has been resolved in {project-short} 2.6.4. link:https://issues.redhat.com/browse/MTV-1299[(MTV-1299)]
```
[the remark this answers is not in the document]
Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc]: We have an entry for MTV-1067 which was the resolution in 2.6.2. 

So I will add a link to that:

.Disks and drives are offline after migrating Windows virtual machines from RHV or VMware to OCP

Windows (Windows 2022) VMs configured with multiple disks, which are *Online* before the migration, are *Offline* after a successful migration from {rhv-short} or VMware to {ocp}, using {project-short}. Only the `C:\` primary disk is *Online*. This issue has been resolved for basic disks in {project-short} 2.6.4. For details of the known issue and resolution for VMs with multiple disks that are migrated from vSphere and OVA files that were not being fully copied, see xref:mtv-1067[(MTV-1067)]. link:https://issues.redhat.com/browse/MTV-1299[(MTV-1299)]

**Review by fabiand — APPROVED** [review]
LGTM!

G as in great

File: documentation/modules/rn-2.6.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc]: Cool, added:

Windows (Windows 2022) VMs configured with multiple disks, which are *Online* before the migration, are *Offline* after a successful migration from {rhv-short} or VMware to {ocp}, using {project-short}. Only the `C:\` primary disk is *Online*. This issue has been resolved for basic disks in {project-short} 2.6.4. link:https://issues.redhat.com/browse/MTV-1299[(MTV-1299)]

For details of the known issue of dynamic disks being *Offline* in Windows Server 2022 after cold and warm migrations from vSphere to container-native virtualization (CNV) with Ceph RADOS Block Devices (RBD), using the storage class `ocs-storagecluster-ceph-rbd`, see xref:mtv-1344[(MTV-1344)].

[the remark this answers is not in the document]
Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc]: [Link working](https://github.com/user-attachments/assets/d8f5b5a7-b8ac-4d52-972f-e3ce17911806)

**Pull request merged**